crates/oxc_linter/src/snapshots/prefer_called_with.snap

```diff
@@ -0,0 +1,27 @@
+---
+source: crates/oxc_linter/src/tester.rs
+assertion_line: 150
+expression: prefer_called_with
+---
+  ⚠ eslint-plugin-jest(prefer-called-with): Suggest using `toBeCalledWith()` or `toHaveBeenCalledWith()`.
+   ╭─[prefer_called_with.tsx:1:1]
+ 1 │ expect(fn).toBeCalled();
+   ·            ──────────
+   ╰────
+  help: Prefer "toBeCalled"With(/* expected args */)
```

Comment:
Is there missing a space between `"toBeCalled"` and `with`?

Reply:
Update. The problem is double quote shouldn't be here. It should be displayed with `toBeCallWith(/* expected args*/)` or `toHaveBeenCallWith(/* expected args*/)`.